### ISSUE TITLE
Reduce DC-Offset caused by the PC Speaker

### ISSUE THREAD

**Issue by rderooy** [joncampbell123/dosbox-x#1670]
**Is your feature request related to a problem? Please describe.**
Another one from dosbox-staging that was just merged, that may be of interest.
https://github.com/dosbox-staging/dosbox-staging/pull/413

**Comment by Wengier** [comment]
Is there a test set to verify if the patch has been correctly applied?

**Comment by kcgen** [comment]
I compared before and with this set of games: https://github.com/dosbox-staging/dosbox-staging/pull/413#issuecomment-637255303

(And Sopwith, when @rderooy suggested it). 

Then I checked non-PC speaker games (SB/GUS/Adlib) that hit the PC speaker for some reason on startup to confirm the DC offset is now quickly being fadded down before it has a chances to impact the mixer (and the other channels).  Jungle Strike is a good example. 

I posted the before and after recordings in the PR.. but yeah, did take a while to step through them.

**Comment by Wengier** [comment]
@kcgen I am glad to see that you are the author of this patch. With the goal of being a complete DOS emulation package, DOSBox-X always welcomes improvements like this, so if possible, could you please add this patch to DOSBox-X too? Thanks!

**Comment by kcgen** [comment]
Hi @Wengier ; thanks for the generous invitation to help DOSBox-X - unfortunately I have little time to put beyond what I'm contributing to dosbox-staging. That said, I'm happy to follow your progress here and add comments when I think I can provide value. Likewise, I'm sure our projects will continue to bear fruit for eachother like we've seen recently.  Cheers :beers:

**Comment by Wengier** [comment]
@kcgen Are you by any chance krcroft, the co-maintainer of the DOSBox-staging project?

**Comment by kcgen** [comment]
Yes (can you drop me an email at my prior username you've mention @gmail.com ?) - and then delete your note above, best regards.

**Comment by Wengier** [comment]
@kcgen Got it.
